doc/install-macosx.md

```diff
@@ -39,6 +39,9 @@ both MacTeX and TeXstudio from the command line. Open a terminal (search
 ```
 brew cask install mactex
 brew cask install texstudio
+after year 2021, these two codes should be as below:
+brew cask install mactex
+brew install --cask mactex
```

Comment:
Thank you for your feedback. You are right, the old command no longer works. I suggest to remove those old command lines and replace them by the new ones instead of adding a comment with the new lines. The option `--cask` is redundant. So the new commands can be reduced:

```
brew install mactex
brew install texstudio
```

or even a single line `brew install mactex texstudio`.

Do you want to update your pull request? Then I can merge it.